src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: '',
+    wheels: 0,
+    version: 0,
```

Comment:
```suggestion
    name: name,
    wheels: wheels,
    version: version,
```
Нам потрібно записувати сюди значення з аргументів.
Або можна використовувати скорочений запис, якщо назва ключа === назві аргумента:
```
{
  name,
  wheels,
  version,
}
```